docs/rules/require-scoped.md

```diff
@@ -56,13 +109,44 @@ Default is set to `always`.
 <style>
 </style>
 
+<style module>
+</style>
+
+<style module="$styles">
+</style>
```

Comment:
There currently isn't a way to disallow both `scoped` and `module`.

I'm wondering which one of these should yield that behavior:
1. `["never"]`
2. `["never", { module: "accept" }]`

I'm leaning towards the second one, in case there needs to be a scenario where a user wants to only disallow `scoped` styles.

What do you think @ota-meshi ?

Reply:
Perhaps the list of current options looks like this:

1. always
2. never
3. always, { module: "accept" }
4. always, { module: "enforce" }
5. never, { module: "enforce" }
6. ?

|                  | 1   | 2   | 3   | 4   | 5   | 6   |
|:-----------------|:----|:----|:----|:----|:----|:----|
| `<style>`        | :x: | :o: | :x: | :x: | :o: | :o: |
| `<style scoped>` | :o: | :x: | :o: | :x: | :o: | :x: |
| `<style module>` | :x: | :o: | :o: | :o: | :x: | :x: |

Looking at this, I found the `never` option is confusing.
I think the behavior of the `never` option should be changed. But I don't want to do this because it is backward compatible.

So what do you think about adding new options such as:

```ts
{
  "vue-scoped-css/require-scoped": ["error", {
    "allows": ("scoped" | "module" | "normal")[]
  }]
}
```

Only style tags specified in `allows` are allowed to be used. Default is `["scoped"]`.
Specify `["scoped", "module"]` to allow `<style scoped>` and `<style module>` . Specify `["module"]` to allow only `<style module>`. Specify `["normal"]` to disallow `<style scoped>` and `<style module>`.


The current `always` and `never` options will be deprecated, but will be supported as backwards compatible.

Reply:
Great idea! I really like that!

I'm wondering if `normal` might be too vague, but don't have any better alternatives. (`unscoped`, `plain`?)

I don't have time to work on this right now but I'll try to revisit it this weekend or next and think about it some more.

Thanks @ota-meshi

Reply:
> I'm wondering if normal might be too vague, but don't have any better alternatives. (unscoped, plain?)

I like `plain` 👍

> I don't have time to work on this right now but I'll try to revisit it this weekend or next and think about it some more.

I'm not in a hurry. I'm happy to have you contribute.